src/resources/js/admin/tickets-emails.js

```diff
@@ -160,30 +159,31 @@ tribe.tickets.emails = {};
 		context.currentEmail = currentEmail;
 
 		if ( currentEmail ) {
-			const fieldReducer = (context, el, fieldPrefix) => {
-				const name = el.name.replace(`${fieldPrefix}-`, '')
+			const fieldReducer = ( ctx, el, fieldPrefix ) => {
+				// eslint-disable-next-line template-curly-spacing
+				const name = el.name.replace( `${fieldPrefix}-`, '' )
 					// From `tec_tickets_emails_current_section-some-field' to `someField`.
-					.replace(/-([a-z])/g, (g) => g[1].toUpperCase()).replace(/\[]$/, '')
-				context[name] = el.type === 'checkbox' ? el.checked : el.value
-				return context
+					.replace( /-([a-z])/g, (g) => g[1].toUpperCase() ).replace( /\[]$/, '' ); // eslint-disable-line space-in-parens, computed-property-spacing, max-len
+				ctx[ name ] = el.type === 'checkbox' ? el.checked : el.value;
```

Comment:
Is there any particular reason for renaming the variable? I believe `context` is more straightforward than `ctx`

Reply:
There is a global variable already names `context`. This resolved any conflicts, plus it resolves the lint errors.